modules/benchmark/DESCRIPTION

```diff
@@ -15,26 +15,29 @@ Description: The Predictive Ecosystem Carbon Analyzer (PEcAn) is a scientific
     streamline the interaction between data and models, and to improve the
     efficacy of scientific investigation.
 Imports:
+    dbplyr,
+    dplyr,
+    ggplot2,
+    gridExtra,
+    lubridate (>= 1.6.0),
+    magrittr,
+    ncdf4 (>= 1.15),
     PEcAn.DB,
     PEcAn.logger,
     PEcAn.remote,
     PEcAn.settings,
     PEcAn.utils,
-    lubridate (>= 1.6.0),
-    ncdf4 (>= 1.15),
-    udunits2 (>= 0.11),
-    XML (>= 3.98-1.4),
-    dplyr,
-    ggplot2,
-    gridExtra,
     reshape2,
-    dbplyr,
     SimilarityMeasures,
-    zoo,
     stringr
```

Comment:
You _do_ need the comma here though. This was must have been accidentally dropped when you re-alphabetized.

```suggestion
    stringr,
```